### PR TITLE
Realization Charges

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Realization engine now requires charges to use.

## Why It's Good For The Game
Realizations now require charges; and you now get one from quota, but eventually you will be able to get 4;
One from Quota, one from Lob, one from Extraction-Cargo and one from Representative.

## Changelog
:cl:
balance: added realization charges
/:cl:

### CODE REVIEW

**Review by EgorDinamit — CHANGES_REQUESTED** [review]
This is poorly implemented right now and basically forces even higher amount of stat rush, preventing "new players" from getting those.

**Comment by Kitsunemitsu** [comment]
> This is poorly implemented right now and basically forces even higher amount of stat rush, preventing "new players" from getting those.

This is just an initial test; I have recently been making rather experimental changes to see how the community thinks about them.

File: code/controllers/subsystem/lobotomy_corp.dm [hunk 1]
```diff
@@ -236,6 +236,8 @@ SUBSYSTEM_DEF(lobotomy_corp)
 			pod.explosionSize = list(0,0,0,0)
 			to_chat(person, "<span class='nicegreen'>It's pizza time!</span>")
 			new /obj/effect/pod_landingzone(get_turf(person), pod)
+
+			GLOB.realization_charges +=1
```
Comment (Gboster-0) [inline code/controllers/subsystem/lobotomy_corp.dm:240]: wont putting this here make dead agents/clerks also give additional charges to the realization engine?
eighter that or the comment at the top is wrong

Comment (Kitsunemitsu) [reply code/controllers/subsystem/lobotomy_corp.dm:240]: Oh wait yeah, it does do that.
I actually got distracted mid-making this PR and hotswapped to doing something cooler.

**Comment by Kitsunemitsu** [comment]
Closed for now.

**Pull request closed without merging**